mpeg-encoder.js

```diff
@@ -0,0 +1,58 @@
+module.exports = function () {
```

Comment:
Unfortunately, we need tests for this file as well.

You can check my test for WAV encoder https://github.com/ai/audio-recorder-polyfill/blob/master/test/wave-encoder.test.js

Reply:
it's really tough to test web worker with importScripts :(

Reply:
Another reason to use `lamejs` from npm

Reply:
OK, another way to test encoder is to emulate `window.importScripts = () => true`

Reply:
do you prefer to add build step for encoders?

Reply:
Nope. It is OK to take `lamejs` from CDN.

I told you about the test for `mpeg-encoder`. You can mock `importScript` and insert fake `window.lamejs` in the test.

Reply:
done